lib/model/Tokens.js

```diff
@@ -95,6 +95,34 @@ Tokens.prototype.connectByApiToken = function(tenantName, apiToken, callback) {
     });
 }
 
+/**
+ * Authenticate with the API endpoint to get the entire token object
+ * 
+ * @param {string} tenantName - name of the tenant we want to access
+ * @param {string} apiToken - the apiToken retrieved from mPulse UI
+ * @param {function} callback - 
+ * [Called after the API Auth request was finalized]{@link Tokens~getToken(callback)}
```

Comment:
I couldn't find the docs for `Tokens~getToken(callback)` and it's not clear from these docs how the callback should be defined or what it should expect.